tools/cockpit.spec

```diff
@@ -249,10 +268,12 @@ autoreconf -fvi -I tools
     --disable-ssh \
 %endif
 
-make -f /usr/share/selinux/devel/Makefile cockpit.pp
-bzip2 -9 cockpit.pp
+make -j4 %{?extra_flags} all
```

Comment:
wrap the above without reformatting so least changes.

%make_build should work with leap.

Reply:
restored

Reply:
fixed